terraform/modules/tf_alert_processor_iam/main.tf

```diff
@@ -62,11 +62,12 @@ data "aws_iam_policy_document" "output_secrets" {
   // Allow retrieving encrypted output secrets
   statement {
     effect    = "Allow"
-    actions   = ["s3:GetObject"]
-    resources = ["arn:aws:s3:::${var.prefix}-streamalert-secrets/*"]
+    actions   = ["ssm:GetParameter"]
+    resources = ["arn:aws:ssm:${var.region}:${var.account_id}:parameter/${var.prefix}_streamalert_secrets/*"]
```

Comment:
if the above "namespace" change is added, this would need changed for the proper path spec

Reply:
I've got a PR in vpc-tf that would need to be updated too